components/FilterSection/index.js

```diff
@@ -36,9 +55,17 @@ export default function FilterSection({
   function onFilter(event) {
     handleFilter(event.target.value, event.target.name);
   }
+  const [isCollapsed, setIsCollapsed] = useState(true);
 
+  function handleExpandFilter() {
+    setIsCollapsed(!isCollapsed);
+  }
   return (
-    <StyledForm>
+    <>
+    <FilterButton onClick={handleExpandFilter}>
+        <FilterIcon />
+      </FilterButton>
+    <StyledForm className={isCollapsed ? "hidden" : ""}>
```

Comment:
Why so complicated? And we don't want to use `className`. Easier:
`{ !isCollapsed && <StyledForm>...</StyledForm> }`